cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -298,6 +298,38 @@ def test_vm_reconfig_resize_disk_hot(disk_type, disk_mode):
     pass
 
 
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_resize_disk_snapshot(disk_type, disk_mode, provider):
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        testtype: functional
+        startsin: 5.11
```

Comment:
This is for polarion - but the test case itself needs a `pytest.mark.ignore_stream('5.10')` to keep the case from being collected against a 5.10 appliance.